src/oatpp/web/server/api/ApiController.hpp

```diff
@@ -485,6 +485,7 @@ template<>
 struct ApiController::TypeInterpretation <oatpp::Int8> {
   static oatpp::Int8 fromString(const oatpp::String& typeName, const oatpp::String& text, bool& success) {
     (void) typeName;
+    //TODO: the below gives warning C4244: 'argument': conversion from 'v_int32' to 'TValueType', possible loss of data
```

Comment:
This is the same case as in the JSON deserializer.
We could probably check that the value actually falls in the range of `Int8`, but it's out-of-scope for now.

Let's just static cast it to silent the error, and leave the corresponding `TODO`